quickwit/quickwit-proto/src/lib.rs

```diff
@@ -464,7 +463,27 @@ pub fn query_ast_from_user_text(user_text: &str, default_fields: Option<Vec<Stri
 // Prost imposes the PartialEq derived implementation.
 // This is terrible because this means Eq, PartialEq are not really in line with Ord's implementation.
 // if in presence of NaN.
+impl PartialOrd for SortBy {
+    fn partial_cmp(&self, other: &SortBy) -> Option<Ordering> {
+        self.sort_value.partial_cmp(&other.sort_value)
+    }
```

Comment:
I think this is derivable no? You can add derive attributes in the prost config.

Reply:
yes